src/Symfony/Component/Validator/Resources/translations/validators.sk.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>Súbor nesmie byť prázdny.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Hostiteľa nebolo možné rozpoznať.</target>
+            </trans-unit>
+            <trans-unit id="80">
+                <source>This value does not match the expected {{ charset }} charset.</source>
+                <target>Táto hodnota nezodpovedá očakávanej znakovej sade {{ charset }}.</target>
```

Comment:
Google translate is wrong.
`zodpovedá` == `match`
`nezodpovedá` == `does not match`. It's teh negative of `zodpovedá`.

I've already contribute a fix to Google translate.